src/plugins/notebook/components/Notebook.vue

```diff
@@ -728,6 +734,39 @@ export default {
             this.selectPage(pageId);
 
             this.syncUrlWithPageAndSection();
+        },
+        activeTransaction() {
+            return this.openmct.objects.getActiveTransaction();
+        },
+        startTransaction() {
+            this.openmct.objects.startTransaction();
+        },
+        saveTransaction() {
+            const transaction = this.activeTransaction();
+
+            if (!transaction) {
+                return;
+            }
+
+            return transaction.commit()
+                .then(() => {
+                // committed
+                }).catch(error => {
+                    throw error;
+                }).finally(() => {
+                    this.openmct.objects.endTransaction();
+                });
+        },
+        cancelTransaction() {
+            return new Promise((resolve, reject) => {
```

Comment:
wondering why do we need to return promise here?

Reply:
we definitely don't need to, I'll remove, it was a holdover from pulling the code from transactions.